exercises/roman-numerals/runtests.jl

```diff
@@ -0,0 +1,36 @@
+using Base.Test
+
+include("roman-numerals.jl")
+
+# NOTE: Got that one from Python's testset
+samples = Dict(
+    1 => "I",
+    2 => "II",
+    3 => "III",
+    4 => "IV",
+    5 => "V",
+    6 => "VI",
+    9 => "IX",
+    27 => "XXVII",
+    48 => "XLVIII",
+    59 => "LIX",
+    93 => "XCIII",
+    141 => "CXLI",
+    163 => "CLXIII",
+    402 => "CDII",
+    575 => "DLXXV",
+    911 => "CMXI",
+    1024 => "MXXIV",
+    3000 => "MMM"
+)
+
+@testset "basic transformations" begin
+    for sample in samples
+        @test to_roman(sample[1]) == sample[2]
+    end
+end
+
+@testset "error handling" begin
+    @test_throws ErrorException to_roman(0)
+    @test_throws ErrorException to_roman(3001)
```

Comment:
I meant something along the lines of "throw an error if `number` is too high for your function to handle". Your hint is already part of the exercise description.

The example solutions aren't downloaded if somebody fetches an exercise through the exercism CLI, so the hint in `example.jl` would not be immediately visible for a user. Instead you can put it in either a new file `HINTS.md` in the same directory as the exercise or put it in the `roman-numerals.jl` code skeleton.

> I think that for zero and negative numbers there should be an error.

I agree. Perhaps add an additional test for negative numbers, too?